pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -181,3 +191,72 @@ func validateExternalClusterSpec(cluster *cluster) error {
 
 	return nil
 }
+
+func (c *ClusterController) configureExternalClusterMonitoring(cluster *cluster) error {
+	// Initialize manager object
+	manager := mgr.New(cluster.Info,
+		c.context,
+		c.namespacedName.Namespace,
+		c.rookImage,
+		cephv1.CephVersionSpec{},
+		rookv1.Placement{},
+		rookv1.Annotations{},
+		cephv1.NetworkSpec{},
+		cephv1.DashboardSpec{},
+		cluster.Spec.Monitoring, // ONLY IMPORTANT PART
```

Comment:
Instead of calling `mgr.New()` it might be simpler to just create the `mgr.Cluster` type directly and only assign the needed fields.

Reply:
If I do this, I'll have to export a bunch of fields from the `Cluster` struct, I wonder if this is worth it. Thoughts?

Reply:
No, in that case not worth it